bfxapi/rest/bfx_rest.py

```diff
@@ -283,6 +283,58 @@ async def get_foreign_exchange_rate(self, ccy1, ccy2):
         message = await self.post(endpoint, payload)
         return message
 
+    async def get_public_stats(self, key, size, symbol, section, side=None,
+                               sort=None, start=None, end=None, limit=None):
+        """
+        The Stats endpoint provides various statistics on a specified trading pair or funding currency.
+
+        # Attributes
+        @param key str
+        Available values -> "funding.size", "credits.size", "credits.size.sym",
+        "pos.size", "vol.1d", "vol.7d", "vol.30d", "vwap"
+
+        @param size str
+        Available values -> "30m", "1d", '1m'
+
+        @param symbol str: the symbol you want information about
+        (e.g. tBTCUSD, tETHUSD, fUSD, fBTC)
+
+        @param section str
+        Available values -> "last", "hist"
+
+        @param side str: only for non-funding queries
+        Available values -> "long", "short"
+
+        @param sort int: if = 1 it sorts results returned with old > new
+
+        @param start str: millisecond start time
+
+        @param end str: millisecond end time
+
+        @param limit int: number of records (max 10000)
+
+        @return
+        Array [MTS, VALUE] with Section = "last"
+        Array [[MTS, VALUE], ...] with Section = "hist"
+        """
+        if key != 'funding.size' and not side:
+            raise Exception('side is compulsory for non funding queries')
+        endpoint = f"stats1/{key}:{size}:{symbol}"
+        if side:
+            endpoint += f":{side}"
+        if section:
+            endpoint += f"/{section}"
+        endpoint += '?'
+        if sort:
+            endpoint += f"sort={sort}&"
```

Comment:
is there something like a url query formatter for python?

else we end up with these `&` signs appended in many cases. example, just sort is passed, urls becomes:

```
stats1/vol.1d:1m:tBTCUSD?sort=1&
```

Reply:
discussed on slack, we will fix those project.wide in the long term